app/src/pages/Play/index.tsx

```diff
@@ -119,6 +119,29 @@ function Play() {
     return null
   }
 
+  const numCompletedCards = completedCardIds.length
+  const totalNumCards = currentGame.cards.length
+  const numRounds = currentGame.rounds.length
+
+  if (numCompletedCards === numRounds * totalNumCards) {
+    return <NoMoreRounds />
+  }
+
+  const roundMarkers = [...Array(numRounds).keys()]
+  let roundMarker = Math.floor(numCompletedCards / totalNumCards)
```

Comment:
I added the `Math.floor` here -- I think this wasn't working right before, but it didn't really have an effect since `roundMarker` wasn't used for much

Reply:
I don't think we want Math.floor here bc we want an exact match (only show the Info Card for the round IF it's 0.0 or 1.0 or 2.0 etc.)

Say you have 10 cards:

First round

0/10 -> 0 (show the info card, first turn of first round)
1/10 -> 0.1

Second round

10/10 -> 1 (show the info card, first turn of second round)
11/10 -> 1.1 

If we use Math.floor it's not correct for displaying the Info Cards, but it IS correct for determining which round we are currently in. Does that makes sense? If so, we might consider two variables for these distinct purposes

Reply:
👍 wondered if it was intentional. Makes sense, I'll revert